docs/usage/package-structure.md

```diff
@@ -0,0 +1,169 @@
+# Important files / folders
+
+## `setup.py`
+The package will follow some conventions, to make it "discoverable" by others.
+
+In `setup.py`, always add the correct Trove Classifiers:
+
+```python
+        "Framework :: Plone",
+        "Framework :: Plone :: 6.0",
+        "Framework :: Plone :: Distribution",
+```
+
+and also require `plone.distribution` to be available:
+
+```python
+    install_requires=[
+        "Plone",
+        "setuptools",
+        "plone.distribution",
+    ],
+```
+
+## `configure.zcml`
+
+In your main `configure.zcml`, make sure to have the `plone` xml namespace declared:
+
+```xml
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    >
+```
+
+And also include `plone.distribution`:
+
+```xml
+  <include package="plone.distribution" />
+```
+
+Then declare the distributions included in your package:
+
+```xml
+
+  <plone:distribution
+      name="blog"
+      title="Personal Blog"
+      description="A Plone site already configured to host a personal Blog."
+      directory="distributions/blog"
+      />
+
+```
+
+The registered distribution will configure a Personal Blog, with some default content.
+
+## `distributions` folder
+
+A convention is to use the `distributions/<distribution_name>`folder in the root of your package to organize your distribution configuration.
+
+In that folder, you will need to provide:
+
+### `image.png`
+
+A 1080x768 image of your distribution. It could be the default page of a new site, your logo, or any other way of representing this distribution.
```

Comment:
```suggestion
A 1080 pixels wide by 768 pixels tall image of your distribution.
It could be the default page of a new site, your logo, or any other way to represent this distribution.
```